compiler_opt/tools/extract_ir.py

```diff
@@ -371,4 +380,5 @@ def main(argv):
 
 
 if __name__ == '__main__':
+  multiprocessing.set_start_method('fork')
```

Comment:
why?

Reply:
oh, meant to include this in the pull request description. it's because of this
https://github.com/abseil/abseil-py/issues/170

basically, the default method of multiprocessing on macos is spawn instead of fork, which means the flags don't get passed around correctly. The flags are accessed within the spawned processes, so without this line macos will use default flag values for anything inside the subprocess.